dbbuild/sources/yu2022g2/Yu2022-TableS2.tsv

```diff
@@ -112,7 +112,7 @@ mh05WL-030	56292990;56293009;56293054;56293077;56293149	rs57258507;rs17732378;rs
 mh05WL-031	146337694;146337725;146337743;146337873	rs7726431;rs143876253;rs13170982;rs17105785	4.20		4	180
 mh05WL-013	62073698;62073771;62073783;62073792;62073794;62073834	rs7715472;rs2642778;rs2591790;rs2591791;rs2642779;rs2642780	4.16		6	137
 mh05WL-033	141129129;141129204;141129208;141129248	rs3797057;rs32946;rs117055966;rs32945	4.13		4	120
-mh06WL-017	32630944;32630945;32630946;32630949;32630967;32630968;32630991;32630995;32630996;32630999;32631000;32631029;32631038;32631039;32631042;32631043;32631044;32631058;32631070;32631075;32631079;32631080;32631086;32631104;32631109;32631113;32631119;32631122;32631129;32631130;32631133;32631142	rs281863504;rs281863503;rs281863502;rs281863499;rs281863486;rs281863485;rs35986240;rs281863467;rs281863466;rs281863464;rs281863463;rs281863439;rs281863431;rs281863430;rs281863427;rs281863426;rs281863425;rs281863414;rs281863406;rs281863401;rs281863398;rs281863397;rs58770498;rs281863382;rs281863378;rs281863375;rs281863371;rs17843723;rs281863364;rs281863363;rs281863362;rs281863357	15.69	6.43	32	199
+mh06WL-017	32630944;32630945;32630946;32630949;32630967;32630968;32630991;32630995;32630996;32630999;32631000;32631029;32631038;32631039;32631042;32631043;32631044;32631058;32631070;32631075;32631079;32631080;32631086;32631104;32631109;32631113;32631119;32631122;32631129;32631130;32631133;32631142	rs281863504;rs281863503;rs281863502;rs281863499;rs281863486;rs281863485;rs35986240;rs281863467;rs281863466;rs281863464;rs281863463;rs281863439;rs281863431;rs281863430;rs281863427;rs281863426;rs281863425;rs281863414;rs281863406;rs281863401;rs281863398;rs281863397;rs17843724;rs281863382;rs281863378;rs281863375;rs281863371;rs17613622;rs281863364;rs281863363;rs281863362;rs281863357	15.69	6.43	32	199
```

Comment:
This is the original change motivating this PR. Digging to the bottom of it took more time than I'd like to admit!